cmd/av/stack_sync.go

```diff
@@ -59,6 +59,10 @@ base branch.
 		if err != nil {
 			return err
 		}
+		if _, err = tidyDB(repo, db); err != nil {
```

Comment:
I think we should print the `Tidied <n> branches` here (though no need to print that we didn't tidy branches,,, so,,, idk) and also add a flag to disable the tidy behavior (`--no-tidy`). 

This is also not blocking.

Reply:
Since tidying is for internal DB to sync up with the git-branches, I feel like it's really internal thing that users shouldn't care about. IDK. Maybe our users do know about av-cli's internal DB and they know what tidy means. In that case, we can surface this and add an option.